spark/src/test/scala/org/apache/spark/sql/delta/deletionvectors/DeletionVectorsSuite.scala

```diff
@@ -334,9 +334,14 @@ class DeletionVectorsSuite extends QueryTest
           val opMetrics = DeltaMetricsUtils.getLastOperationMetrics(tableName)
           assert(opMetrics.getOrElse("numDeletedRows", -1) === 1)
           assert(opMetrics.getOrElse("numRemovedFiles", -1) === 0)
-          assert(opMetrics.getOrElse("numDeletionVectorsAdded", -1) === 0)
-          assert(opMetrics.getOrElse("numDeletionVectorsRemoved", -1) === 0)
-          assert(opMetrics.getOrElse("numDeletionVectorsUpdated", -1) === 1)
+          val initialNumDVs = 0
+          val numDVUpdated = 1
+          // An "updated" DV is "deleted" then "added" again.
+          // We increment the count for "updated", "added", and "deleted".
+          val numDVAddedOrRemoved = initialNumDVs + numDVUpdated
+          assert(opMetrics.getOrElse("numDeletionVectorsAdded", -1) === numDVAddedOrRemoved)
+          assert(opMetrics.getOrElse("numDeletionVectorsRemoved", -1) === numDVAddedOrRemoved)
```

Comment:
nit: the name `numDVAddedOrRemoved` is confusing. Just use the `numDVUpdated` or create separate vars for each add and delete.